### PR TITLE
convert into single package

### PR DESCRIPTION
**Pull request by angusfretwell**

### CODE REVIEW

File: .buildkite/docker-compose.yml [hunk 1]
```diff
@@ -10,13 +10,14 @@ services:
...
-      CI: "true"
-      TZ: "Australia/Sydney"
+      CI: 'true'
+      TZ: 'Australia/Sydney'
+      APPLICATION:
       BUILDKITE_BUILD_CREATOR_EMAIL:
       BUILDKITE_BUILD_CREATOR:
       BUILDKITE_BUILD_NUMBER:
       BUILDKITE_COMMIT:
+      GITHUB_TOKEN:
```
Comment (allanpope) [inline .buildkite/docker-compose.yml]: Whats the github token needed for?

File: .buildkite/pipeline.yml [hunk 2]
```diff
@@ -28,20 +23,18 @@ steps:
...
   - wait
 
-  - name: ":npm: Publish"
-    <<: *deploy_defaults
-    command: scripts/publish
-
-  - name: ":storybook: Deploy"
-    <<: *deploy_defaults
-    command: yarn deploy-storybook
+  - name: ':npm: Release'
```
Comment (allanpope) [inline .buildkite/pipeline.yml]: You'll only want to do this on the master branch

File: .buildkite/pipeline.yml [hunk 3]
```diff
@@ -28,20 +23,18 @@ steps:
...
     <<: *defaults
     command: yarn run test
 
   - wait
 
-  - name: ":npm: Publish"
-    <<: *deploy_defaults
-    command: scripts/publish
-
-  - name: ":storybook: Deploy"
```
Comment (allanpope) [inline .buildkite/pipeline.yml]: Not deploying storybook anymore?

File: .eslintrc.js [hunk 4]
```diff
@@ -0,0 +1,49 @@
+module.exports = {
+  parser: 'babel-eslint',
+  env: {
+    browser: true,
+    node: true,
+    es6: true,
+  },
+  settings: {
+    react: {
+      version: '16.9',
```
Comment (allanpope) [inline .eslintrc.js]: Why do you need to specify this?

File: .buildkite/pipeline.yml (hunk 2 above)
Comment (angusfretwell) [reply .buildkite/pipeline.yml]: Nah, semantic release will handle this now. It supports releasing from other branches like alpha or beta.

File: .buildkite/docker-compose.yml (hunk 1 above)
Comment (angusfretwell) [reply .buildkite/docker-compose.yml]: `semantic-version` needs it to write tags/releases to Github.

image

File: .buildkite/pipeline.yml (hunk 3 above)
Comment (angusfretwell) [reply .buildkite/pipeline.yml]: It was being deployed to Github pages as well as Netlify (for PRs). Decided to drop Github pages and just point to the Netlify deployment instead, so we don't have 2 different ways of deploying.

File: .eslintrc.js (hunk 4 above)
Comment (angusfretwell) [reply .eslintrc.js]: For `eslint-plugin-react`. It shows a warning when running the linter otherwise.

**Pull request merged**